src/main/java/org/eaa690/aerie/communication/SendGridEmailSender.java

```diff
@@ -0,0 +1,36 @@
+package org.eaa690.aerie.communication;
+
+import java.io.IOException;
+
+import com.sendgrid.Method;
+import com.sendgrid.Request;
+import com.sendgrid.Response;
+import com.sendgrid.SendGrid;
+import com.sendgrid.helpers.mail.Mail;
+
+import org.eaa690.aerie.service.PropertyService;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class SendGridEmailSender implements MessageSender {
```

Comment:
This is effectively the same as the existing EmailService.  Simply modify the EmailService with the MessageSender interface and modify the EmailService class to adhere to the interface